src/generators/openapi/generators/openapi-utils/convertApiBuilderPrimitiveType.ts

```diff
@@ -11,7 +11,16 @@ function convertApiBuilderPrimitiveType(type: ApiBuilderPrimitiveType) {
     case Kind.DECIMAL: return { type: 'number', format: 'float' };
     case Kind.DOUBLE: return { type: 'number', format: 'double' };
     case Kind.INTEGER: return { type: 'integer' };
-    case Kind.JSON: return { AnyValue: {} };
+    case Kind.JSON: return {
```

Comment:
Feel like your branches are out of sync. This change was already merged. Can you fix branch and re-open?